src/pages/creations/index.tsx

```diff
@@ -33,9 +30,16 @@ export default function Creations() {
               <BG_3 type='2' align='right' />
             </BackgroundContainer>
             <HeroDivider>
-              <TitleVideo>
-                <AnimatedTitle chromeSrc={VIDEO_CHROME} safariSrc={VIDEO_SAFARI} />
-              </TitleVideo>
+              <SquigglyTitle
```

Comment:
this title should be more centered in desktop and mobile views

before, 
<img width="1398" alt="image" src="https://github.com/user-attachments/assets/2f221b59-f77c-42ff-92ed-70f8478bdf26">
after,
<img width="1398" alt="image" src="https://github.com/user-attachments/assets/92e82b8b-b926-4009-b085-6be41755a597">

Reply:
[6697cf9](https://github.com/defi-wonderland/web/pull/110/commits/6697cf92e720fffa1e14803fa4c31bec9226dd83)